Keas.Mvc/Controllers/Api/AccessController.cs

```diff
@@ -75,19 +75,43 @@ public async Task<IActionResult> List()
 
             return Json(accessList);
         }
+
+        /// <summary>
+        /// Lists all Access items that have been deleted for the team
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Access>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> ListInactive()
```

Comment:
@jSylvestre I think you are probably better off here doing an overload like you did in the report controller.  Just add an param to our existing `List()` method that defaults `includeInactive == false`.

Reply:
@srkirkland I'm ok with that, but I was thinking it would be useful to be able to see just inactive stuff.